pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -94,6 +94,18 @@ func (r *ReconcileClusterDeployment) Reconcile(request reconcile.Request) (recon
 		return reconcile.Result{}, err
 	}
 
+	// Do not make certificate request if the cluster is not a Red Hat managed cluster.
+	if val, ok := cd.Labels["api.openshift.com/managed"]; ok {
+		if val != "true" {
+			reqLogger.Info("Not a managed cluster", "Namespace", request.Namespace, "Name", request.Name)
```

Comment:
here and in the other logger line below it adds the 'Namespace' and 'Name' fields to the log output, but the very first line in the Reconcile() func already set up those fields (with different names 'Request.Namespace' and 'Request.Name')

Reply:
updated